socli/socli.py

```diff
@@ -361,6 +362,44 @@ def main():
         search.google_search = False
         tag = namespace.tag
         has_tags()  # Adds tags to StackOverflow url (when not using google search.
+    if namespace.open_url:
+        import webbrowser
+        display_condition=True
+        url_to_use=namespace.open_url[0]
+        nostackoverflow=re.search("stackoverflow",url_to_use)
+        if nostackoverflow:
+            pass
+        else:
+            printer.print_fail("Your url is not a stackoverflow url.\n Opening in your browser")
+            raise Exception("Given url is not a stackoverflow url")
```

Comment:
I don't think we need to raise exception here? because you have already printed the message on the line above